src/wallet/sqlite.h

```diff
@@ -36,11 +36,21 @@ class SQLiteCursor : public DatabaseCursor
     Status Next(DataStream& key, DataStream& value) override;
 };
 
+/** Abstract class responsible for executing SQL statements in SQLite databases.
+ *  It is abstract so it can be overridden by unit tests testing unusual database conditions. */
```

Comment:
In commit "sqlite: add ability to interrupt statements" (177ff954c1221a00d4bf5ea63e6bfc415ba706c8)

It's technically not an abstract class anymore since it doesn't contain any `= 0` pure virtual methods. Would maybe suggest "Class responsible for executing SQL statements in SQLite databases. Methods are virtual so they can be overridden by unit tests testing unusual database conditions."

Reply:
yeah, fixed. I pasted your doc suggestion first, then made the classes consolidation..